sdk/eventgrid/azure-eventgrid/samples/consume_samples/e2e_samples/functionsapp/.funcignore

```diff
@@ -0,0 +1,5 @@
+.git*
+.vscode
+local.settings.json
+test
+.venv
```

Comment:
do we want this to be the .funcignore everyone has? it seems a tiny bit special cased (ala .venv/test, but that may be intentional)